scipy/stats/_continuous_distns.py

```diff
@@ -5800,6 +5800,12 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _fitstart(self, data, args=None):
+        # Empirically good first estimates
```

Comment:
Is there a reference for this, or by "empirically" do you mean in informal experiments?

Reply:
You are right. I've included a reference to the original issue in 8dd8d55668899160bab065ebc89e1d9e2a2a4638.